### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by radioblahaj** [Pufflegamerz/pufflegamerz.github.io#1]

**Comment by petabyt** [comment]
Hello

**Comment by petabyt** [comment]
https://github.com/Pufflegamerz/ScratchFormat/issues/1

**Comment by radioblahaj** [comment]
Hi!

**Comment by radioblahaj** [comment]
<a href="https://github.com/Pufflegamerz/ScratchFormat/issues/1" class="issue-link js-issue-link tooltipped tooltipped-ne" data-error-text="Failed to load issue title" data-id="278826788" data-permission-text="Issue title is private" aria-label="Pufflegamerz/ScratchFormat#1, Image not working with text beside it">Pufflegamerz/ScratchFormat#1</a>
